_content/booklets/the-mind-and-its-endless-rebirth_suchart.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "The Mind and its Endless Rebirth"
+authors:
+  - suchart
+external_url: "http://www.kammatthana.com/dt_011.pdf"
```

Comment:
🤔 Unfortunately this link won't work given Chrome's new security features. Let's just make it `source_url` and leave the drive link below as the primary link.